modules/ballerina-http/src/main/ballerina/ballerina/net/http/natives.bal

```diff
@@ -1,16 +1,36 @@
 package ballerina.net.http;
 
-@Description { value:"Represents an HTTP request message"}
-@Field {value:"host: The server host name"}
+@Description { value:"Represents the HTTP server connection"}
+@Field {value:"remoteHost: The server host name"}
 @Field {value:"port: The server port"}
+public struct Connection {
+	string remoteHost;
+	int port;
+}
+
+@Description { value:"Sends response to the caller"}
+@Param { value:"conn: The server connector connection" }
```

Comment:
Same concern as the previous comment on the terminology

Reply:
keeping the same description as user can get the rough idea about the conn with it.